contracts/ColonyFunding.sol

```diff
@@ -388,4 +390,15 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs {
 
     updateTaskPayoutsWeCannotMakeAfterBudgetChange(_id, _token, currentTotalAmount);
   }
+
+  function calculateNetworkFeeForPayout(uint256 _payout) private view returns (uint256 fee) {
+    IColonyNetwork colonyNetworkContract = IColonyNetwork(colonyNetworkAddress);
+    uint256 feeInverse = colonyNetworkContract.getFeeInverse();
+
```

Comment:
With this ☝️ if they are both 2 we get a fee of 2 which means effective fee of 100% rather than the intended 50%, that error margin decreases as the values increase, i.e. if they are both 100 we get an effective fee of 2% rather than intended 1% etc. We are slightly overcharging using this calculation as opposed to slightly undercharging using mine above.

Reply:
With this ☝️ if they are both 2 we get a fee of 2 which means effective fee of 100% rather than the intended 50%, that error margin decreases as the values increase, i.e. if they are both 100 we get an effective fee of 2% rather than intended 1% etc. We are slightly overcharging using this calculation as opposed to slightly undercharging using mine above.

Reply:
With this ☝️ if they are both 2 we get a fee of 2 which means effective fee of 100% rather than the intended 50%, that error margin decreases as the values increase, i.e. if they are both 100 we get an effective fee of 2% rather than intended 1% etc. We are slightly overcharging using this calculation as opposed to slightly undercharging using mine above.

Reply:
With this ☝️ if they are both 2 we get a fee of 2 which means effective fee of 100% rather than the intended 50%, that error margin decreases as the values increase, i.e. if they are both 100 we get an effective fee of 2% rather than intended 1% etc. We are slightly overcharging using this calculation as opposed to slightly undercharging using mine above.

Reply:
With this ☝️ if they are both 2 we get a fee of 2 which means effective fee of 100% rather than the intended 50%, that error margin decreases as the values increase, i.e. if they are both 100 we get an effective fee of 2% rather than intended 1% etc. We are slightly overcharging using this calculation as opposed to slightly undercharging using mine above.

Reply:
We agreed to take Alex's calculation as that slightly favours the network in terms of error margins in fee calculations as opposed to mine which is slightly more in favour of the client. This has been implemented.